WcaOnRails/app/models/registration.rb

```diff
@@ -86,6 +83,10 @@ def best_solve(event, type)
     person ? person.best_solve(event, type) : SolveTime.new(event.id, type, 0)
   end
 
+  def event_picker_events
+    registration_events.map(&:event_object).sort_by(&:rank)
+  end
```

Comment:
@jonatanklosko - I originally did  use `registration.events` but I found that if there was a validation error, for example you left the number of guests as blank and selected a new event, then when the form reloaded, it had lost the extra event. I think this is because the events_picker only populates RegistrationEvents (and CompetitionEvents). When this gets saved to the database, then you can access `Registration.events`